.github/workflows/php-unit-tests.yml

```diff
@@ -30,11 +30,11 @@ jobs:
       WP_TESTS_DIR: "/tmp/wordpress/tests/phpunit"
     strategy:
       matrix:
-        php: [7.4]
-        wp-version: [5.8, 5.9, latest]
+        php: [7.4, 8.0, 8.1, 8.2]
+        wp-version: [latest]
         include:
-          - php: 8.0
-            wp-version: latest
+          - php: 7.4
+            wp-version: 5.9
```

Comment:
❓ Why is this 5.9? The latest WP version is 6.3. If let's say we follow "L-2" policy, then the version here should be 6.1, no? 🤔

Reply:
The idea was to preserve one old check PHP7.4 + WP5.9 from the previous setup. So if any new code doesn't work with this setup, we would know what is going wrong.

>WooCommerce now uses L-1 support policy.
I think it is fine to use latest wp-version, so that we can catch issues early, instead of using older wp-version.

Given that supporting older versions isn't within our scope and everything works fine up until this point, we are good to remove these and just have the latest WordPress as you suggest.

Reply:
Fixed in [this commit](https://github.com/woocommerce/facebook-for-woocommerce/pull/2624/commits/b8f3cb9e3a5be2080349bdfef0a135cfac1f0403).